x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form_consumer_selection.tsx

```diff
@@ -67,13 +67,20 @@ export interface RuleFormConsumerSelectionProps {
   consumers: RuleCreationValidConsumer[];
   onChange: (consumer: RuleCreationValidConsumer | null) => void;
   errors: IErrorObject;
-  selectedConsumer: RuleCreationValidConsumer | null | undefined;
+  selectedConsumer?: RuleCreationValidConsumer | null;
+  initialSelectedConsumer?: RuleCreationValidConsumer | null;
 }
 
 const SINGLE_SELECTION = { asPlainText: true };
 
 export const RuleFormConsumerSelection = (props: RuleFormConsumerSelectionProps) => {
-  const { consumers, errors, onChange, selectedConsumer } = props;
+  const {
+    consumers,
+    errors,
+    onChange,
+    selectedConsumer,
+    initialSelectedConsumer = STACK_ALERTS_FEATURE_ID,
```

Comment:
Does it mean we cannot set this field to `undefined` or `null` in order to ask the user to set this field?

Reply:
yes, I will put it back the way it was just wanted to avoid the the hard coded condition.